packages/vats/src/bootstrap.js

```diff
@@ -198,7 +198,11 @@ export function buildRootObject(vatPowers, vatParameters) {
 
     // Now we can bootstrap the economy!
     const bootstrapPaymentValue = Nat(BigInt(centralBootstrapSupply.amount));
-    const treasuryCreator = await installEconomy(bootstrapPaymentValue);
+    // NOTE: no use of the voteCreator. We'll need it to initiate votes on
+    // changing Treasury parameters.
+    const { treasuryCreator, _voteCreator } = await installEconomy(
```

Comment:
If you keep the committee in place, maybe you could make it just 1 member, and send the `voteCreator` to the some place as the `treasuryCreator` is exposed (look for `powerFlagConfig`) and name it `treasuryVoteCreator`.